src/ServicePulse.Host/app/js/views/failed_groups/view.html

```diff
@@ -56,10 +56,10 @@
                                         <button type="button" class="btn btn-link btn-sm hidden-sm-down" title="View messages in this Group" ng-click="vm.viewExceptionGroup(group)" ng-disabled="group.count == 0">
                                             View Messages
                                         </button>
-                                        <button type="button" class="btn btn-link btn-sm" title="Retry messages in this Group" confirm-click="vm.retryExceptionGroup(group)" ng-disabled="group.count == 0 || group.workflow_state.status !== 'ready'" confirm-message="Are you sure you want to retry this group of {{group.count}} messages?">
+                                        <button type="button" class="btn btn-link btn-sm" title="Retry messages in this Group" confirm-click="vm.retryExceptionGroup(group)" ng-disabled="group.count == 0 || group.workflow_state.status !== 'ready'" confirm-title="Are you sure you want to retry this group?" confirm-message="Are you sure you want to retry this group of {{group.count}} messages?">
                                             Retry Group
                                         </button>
-                                        <button type="button" class="btn btn-link btn-sm" title="Archive messages in this Group" confirm-click="vm.archiveExceptionGroup(group)" ng-disabled="group.count == 0 || group.workflow_state.status !== 'ready'" confirm-message="Are you sure you want to archive this group of messages? If you do, they will not be available for Retry.">
+                                        <button type="button" class="btn btn-link btn-sm" title="Archive messages in this Group" confirm-click="vm.archiveExceptionGroup(group)" ng-disabled="group.count == 0 || group.workflow_state.status !== 'ready'" confirm-title="Are you sure you want to archive this group?" confirm-message="Are you sure you want to archive this group of messages? If you do, they will not be available for Retry.">
```

Comment:
Any special notes, concerns or consequences the user should be aware of before deciding to archive?